bay/images/Dockerfile.node

```diff
@@ -0,0 +1,42 @@
+FROM uselagoon/node-14:20.12.0
+
+ENV CYPRESS_INSTALL_BINARY 0
+ENV PUPPETEER_SKIP_CHROMIUM_DOWNLOAD 1
+
+# Build time env vars
+ONBUILD ARG CONTENT_API_SERVER
+ONBUILD ARG CONTENT_API_AUTH_PASS
+ONBUILD ARG CONTENT_API_AUTH_USER
+ONBUILD ARG CONTENT_API_OAUTH_CLIENT_ID
+ONBUILD ARG BASIC_AUTH
+ONBUILD ARG TIDE_DEBUG
+ONBUILD ARG DISPLAY_ERROR
+ONBUILD ARG GTM_ID
+ONBUILD ARG SEARCH_HASH
+ONBUILD ARG SEARCH_SERVICE
+ONBUILD ARG SEARCH_INDEX
+ONBUILD ARG SEARCH_URL
+ONBUILD ARG SEARCH_LOG
+ONBUILD ARG SEARCH_AUTH_USERNAME
+ONBUILD ARG SEARCH_AUTH_PASSWORD
+ONBUILD ARG LAGOON_ENVIRONMENT_TYPE
+
+ONBUILD ENV CONTENT_API_SERVER=$CONTENT_API_SERVER
+ONBUILD ENV CONTENT_API_AUTH_PASS=$CONTENT_API_AUTH_PASS
+ONBUILD ENV CONTENT_API_AUTH_USER=$CONTENT_API_AUTH_USER
+ONBUILD ENV CONTENT_API_OAUTH_CLIENT_ID=$CONTENT_API_OAUTH_CLIENT_ID
+ONBUILD ENV BASIC_AUTH=$BASIC_AUTH
+ONBUILD ENV TIDE_DEBUG=$TIDE_DEBUG
+ONBUILD ENV DISPLAY_ERROR=$DISPLAY_ERROR
+ONBUILD ENV GTM_ID=$GTM_ID
+ONBUILD ENV SEARCH_HASH=$SEARCH_HASH
+ONBUILD ENV SEARCH_SERVICE=$SEARCH_SERVICE
+ONBUILD ENV SEARCH_INDEX=$SEARCH_INDEX
+ONBUILD ENV SEARCH_URL=$SEARCH_URL
+ONBUILD ENV SEARCH_LOG=$SEARCH_LOG
+ONBUILD ENV SEARCH_AUTH_USERNAME=$SEARCH_AUTH_USERNAME
+ONBUILD ENV SEARCH_AUTH_PASSWORD=$SEARCH_AUTH_PASSWORD
+ONBUILD ENV LAGOON_ENVIRONMENT_TYPE=$LAGOON_ENVIRONMENT_TYPE
+# End of build time env vars
+
+RUN apk --update add curl
```

Comment:
Nice. We should also remove curl installation from project-specific dockerfiles.

Reply:
@tim-yao's going to roll out updates to all of the projects which will take care of that.